src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/DebugModeType.php

```diff
@@ -43,7 +43,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('disable_non_native_modules', SwitchType::class, [
                 'required' => false,
-                'label' => $this->trans('Disable non PrestaShop modules', 'Admin.Advparameters.Feature'),
+                'label' => $this->trans('Disable non native-in modules', 'Admin.Advparameters.Feature'),
```

Comment:
I may be crazy but I don't understand "native-in". Am I too old?

Isn't it 'Disable non native modules' rather ? or 'Disable non initial/original modules' ?

Reply:
I see it comes from issue https://github.com/PrestaShop/PrestaShop/issues/9965. I ask wording manager Julie.

Reply:
```suggestion
                'label' => $this->trans('Disable non built-in modules', 'Admin.Advparameters.Feature'),
```

Reply:
I think it was a mix of "native" and "built-in". You're not crazy, haha. :-)